demo/c/README.md

```diff
@@ -146,28 +148,30 @@ demo\\c\\build\\cheetah_demo.exe ^
 ```console
 ./demo/c/build/cheetah_demo \
 -a ${ACCESS_KEY} \
--l lib/jetson/cortex-a57-aarch64/libpv_cheetah.so \
 -m lib/common/cheetah_params.pv \
+-l lib/jetson/cortex-a57-aarch64/libpv_cheetah.so \
 -i ${MIC_INDEX}
 ```
 
 #### File Demo
 
 ```console
-usage: -a ACCESS_KEY -l LIBRARY_PATH -m MODEL_PATH wav_path0 wav_path1 ...
+usage: -a ACCESS_KEY -m MODEL_PATH -l LIBRARY_PATH [-p] wav_path0 wav_path1 ...
 ```
 
 Run the command corresponding to your platform from the root of the repository. Replace `${ACCESS_KEY}` with yours
-obtained from [Picovoice Console](https://console.picovoice.ai/) and `${WAV_PATH}` with the path to a compatible 
+obtained from [Picovoice Console](https://console.picovoice.ai/) and `${WAV_PATH}` with the path to a compatible
 (single-channel, 16 kHz, and 16-bit PCM) WAV file you want to transcribe.
 
+Use the `-p` flag to disable automatic punctuation.
```

Comment:
using `-p` to disable automatic punctuation is a bit counter intuitive, maybe `-d`